game/src/main/kotlin/gg/rsmod/game/model/entity/Player.kt

```diff
@@ -235,6 +235,11 @@ open class Player(world: World) : Pawn(world) {
      */
     var lastMapBuildTime = 0
 
+    /**
+     * Web fatigue, used for cutting through spider webs.
+     */
```

Comment:
This can be an attribute instead of a Player var, find attributes in `Attributes.kt`. 

It's better to use the attribute system for content such as the webs. Player vars should be for core things only (skills, inventories, maps, etc)